xgo.go

```diff
@@ -304,19 +308,28 @@ func compile(image string, config *ConfigFlags, flags *BuildFlags, folder string
 		"-e", fmt.Sprintf("FLAG_BUILDMODE=%s", flags.Mode),
 		"-e", "TARGETS=" + strings.Replace(strings.Join(config.Targets, " "), "*", ".", -1),
 	}
-	for i := 0; i < len(locals); i++ {
-		args = append(args, []string{"-v", fmt.Sprintf("%s:%s:ro", locals[i], mounts[i])}...)
-	}
-	args = append(args, []string{"-e", "EXT_GOPATH=" + strings.Join(paths, ":")}...)
-
 	if usesModules {
 		args = append(args, []string{"-e", "GO111MODULE=on"}...)
+		args = append(args, []string{"-v", os.Getenv("GOPATH") + ":/go"}...)
+
+		// Map this repository to the /source folder
+		absRepository, err := filepath.Abs(config.Repository)
```

Comment:
Please add an error check.

Reply:
We can safely ignore `err` because we've made this call once before and have handled the error at: https://github.com/kolaente/xgo/blob/c75d334c67fe18c14242a101b46853d6a7931331/xgo.go#L382 That said, I'm happy to add it in as a defensive measure.